blocks/cards/cards.css

```diff
@@ -261,11 +269,21 @@ main .cards-container .cards-wrapper {
     bottom: 0;
   }
 
+  .meet-our-team-desktop-view ul li > div p {
+    font-style: normal;
+    text-align: center;
+  }
+
   .review > ul > li .cards-card-body p {
-    font-style: italic;
     font-size: 14px;
     margin: 2px 0;
-    line-height: 25px;
+    line-height: 22px;
+    padding: 0 15px;
+  }
+
+  .review > ul > li .cards-card-body p:nth-child(2),
+  .review > ul > li .cards-card-body p:nth-child(3) {
```

Comment:
you should prefer to use semantics rather than accessing p tags via position numbers